src/js/selection/SelectionManager.js

```diff
@@ -91,7 +93,29 @@
     }
   };
 
-  ns.SelectionManager.prototype.paste = function() {
+  // Iterate through pixels to get the largest x and y values of a selection.
+  ns.SelectionManager.prototype._getBoundaryEdges = function(pixels) {
```

Comment:
Can I suggest some different naming here?  You're kind of working out the lower-right corner of your selection's bounding box, and 'edges' may not express that well enough.  I'd almost suggest generalizing this function to find and return a bounding box for the set of pixels and name it that, then use `bottom` and `right` properties on the returned object.  Or maybe just `getBottomRightCorner`?